### DISCUSSION TITLE
MIDISystemInfo DetailView replacement?

### DISCUSSION THREAD

**Discussion by SinanKarasu** [orchetect/swift-midi#165] [General]
To get rid of the WebKit, I've coded it as follows.If you're interested.
[DetailsView.swift.zip](https://github.com/orchetect/MIDIKit/files/10738114/DetailsView.swift.zip)
Some minor changes to the ContentView
[MIDISystemInfoView.swift.zip](https://github.com/orchetect/MIDIKit/files/10738117/MIDISystemInfoView.swift.zip)

**Comment by orchetect** [comment]
Thanks for posting. That demo has to maintain backwards compatibility with macOS 10.15 which is why it has two methods for rendering tables. Arguably WebKit probably wasn't needed in the first place and legacy views could have been done with a mess of Stacks.

**Reply by orchetect** [reply]
I did a major refactor on MIDISystemInfo and the code is much more streamlined. It also runs on both macOS and iOS now.

Pushed to main branch.
